automation/test_env.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Values: start|end
+command=$1
+
+if [[ "$command" = "start" ]]; then
+	docker build -t lioramilbaum/ubuntu .
```

Comment:
do I really need to rebuild the container every time? Did you check if this can skip some build steps because of container layering?

Reply:
the build will skip those layers which are identical to the one in the latest image.